package.json

```diff
@@ -40,6 +41,7 @@
     "@blockone/tslint-config-blockone": "3.0.0",
     "@types/jest": "24.9.1",
     "@types/node": "11.9.4",
+    "@types/pako": "^1.0.1",
     "cypress": "3.8.2",
     "eosjs-ecc": "^4.0.7",
```

Comment:
Should `@types/pako` also be version locked?

And probably unrelated to this PR, but can we remove `eosjs-ecc` here?

Reply:
It doesn't seem like devDependencies have usually been locked but I can update it.

`eosjs-ecc` is still there for the `eosjs-ecc-verification.test.js` file that confirms that `elliptic` and `eosjs-ecc` match, but I don't think we will need it on release.  Let me know if you'd like me to remove them in this PR or another one.

Reply:
We can leave it in this PR for now then, and figure out with the team when is an appropriate time to remove it.